pkg/monitoring/services/services.go

```diff
@@ -0,0 +1,290 @@
+// +build !windows
+
+package services
+
+import (
+	"bufio"
+	"bytes"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"regexp"
+	"runtime"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// ErrorNotImplementedForOS returned in case we don't yet implement service manager parsing or this OS. Should be checked and ignored
+var ErrorNotImplementedForOS = errors.New("Services list not implemented for " + runtime.GOOS)
+
+// SystemdService contains the service's data parsed from systemctl
+type SystemdService struct {
+	UnitFile    string
+	LoadState   string
+	ActiveState string
+	SubState    string
+	Description string
+}
+
+// SysVService contains the service's data parsed from service(Sysvinit) or initctl(Upstart)
+type SysVService struct {
+	UnitFile string
+	Status   string
+}
+
+// ListSystemdServices list Systemd services via systemctl
+func ListSystemdServices() ([]SystemdService, error) {
+	cmd := exec.Command("systemctl",
```

Comment:
We already had issues with parsing the output of local binaries. Is it "safe" to do this way?

Reply:
> Before we had some issues because of columns parsing, now I make it more robust because it reads the header first to match the values

👍

An idea (out of scope if this PR):
Should we think about a feature, that whenever we fail to parse some output like this, we collect and send it to our servers to we can analyse what went wrong?

Reply:
Yes, it can be useful, but
- We will probably need some log processing setup like logstash
- I think it's better for @thorstenkramm to decide if this is OK in terms of privacy(we only send what absolutely necessary)

Reply:
yeah sure, I wasn't thinking about to add this here and now